nbs/08_vision.data.ipynb

```diff
@@ -470,7 +479,16 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
Let's remove this warning because it creates a big diff depending on which OS is used to run the NBs.

Reply:
Thanks reminder.
Cleared all outputs manully.